string.go

```diff
@@ -30,40 +28,45 @@ import (
 	"unicode/utf8"
 )
 
-// AESEncrypt encrypts text and given key with AES.
-func AESEncrypt(key, text []byte) ([]byte, error) {
+// AESEncrypt encrypts plaintext with the given key and nonce using AES in GCM mode.
+func AESEncrypt(key, nonce, plaintext []byte) ([]byte, error) {
```

Comment:
I changed the signature to take in a nonce because the nonce (or IV if you want to call it that) has to be 100% unique. For example, if you go to encrypt a piece of data the encryption/decryption for that piece of data should use its own randomly generated nonce. If you go to encrypt N other pieces of data the encryption/decryption for each piece of data should use its own randomly generated nonce.